app/ore/project/Dependency.scala

```diff
@@ -40,4 +40,8 @@ object Dependency {
     */
   val ForgeId: String = "forge"
 
+  val SpongeForgeId: String = "spongeforge"
```

Comment:
Problem: There is no `spongeforge`/`spongevanilla` plugin ID. Both implementations use `sponge` as plugin ID currently.